firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/NewReleaseFetcher.java

```diff
@@ -129,8 +125,7 @@ boolean isSameAsInstalledRelease(AppDistributionReleaseInternal newRelease)
 
```

Comment:
Interesting idea. I'm having a hard time thinking of a way this could be simpler without sacrificing some of the detailed error messages in various failure cases, though, so I'm inclined to leave it as it is for now.